packages/orderbook/test/js_api/gui.test.ts

```
@@ -409,23 +426,26 @@ describe('Rain Orderbook JS API Package Bindgen Tests - Gui', async function ()
+		const gui = new DotrainOrderGui();
 		mockServer
 			.forPost('/rpc-url')
 			.withBodyIncluding('0x82ad56cb')
 			.thenSendJsonRpcResult(
 				'0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000003000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001a0000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000020000000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000007546f6b656e203100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000060000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000025431000000000000000000000000000000000000000000000000000000000000'
 			);
-		const gui = await DotrainOrderGui.chooseDeployment(dotrainWithGui, 'some-deployment');
-		const deploymentDetail: NameAndDescriptionCfg = gui.getCurrentDeploymentDetails();
+		await gui.chooseDeployment(dotrainWithGui, 'some-deployment');
+
+		let deploymentDetail = extractWasmEncodedData<NameAndDescriptionCfg>(
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'deploymentDetail' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)